state.go

```diff
@@ -13,4 +13,6 @@ type State struct {
 	Pid int `json:"pid"`
 	// BundlePath is the path to the container's bundle directory.
 	BundlePath string `json:"bundlePath"`
+	// Annotations holds runtime-specified information that is not structured by this specification.
```

Comment:
On Mon, Nov 16, 2015 at 08:01:42PM -0800, Vish Kannan wrote:
> > + // Annotations holds runtime-specified information that is not
> >      structured by this specification.
> 
> Why are we mentioning `runtime` here? Are we precluding humans from
> ever trying to use this Spec :smiley: ?

No, we're reserving this namespace *for the runtime to define*.  So
anyone is welcome to read it, but the runtime is the ultimate
authority on any specification for the content, and the only one who
should be writing to it.  Otherwise we'd need sub-namespaces to keep
different players separate (e.g. see [1], but Gao pointed out that we
didn't need a state JSON entry for the bundle author [2]).

[1]: https://github.com/opencontainers/specs/pull/188#issuecomment-146912525
[2]: https://github.com/opencontainers/specs/pull/188#issuecomment-147026995